CipherText/Rotator/Store/StoreRegistryBuilder.php

```diff
@@ -0,0 +1,114 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: jjose00
+ * Date: 11/16/17
+ * Time: 1:26 PM
+ */
+
+namespace Giftcards\Encryption\CipherText\Rotator\Store;
+
+use Giftcards\Encryption\Factory\BuilderInterface;
+use Giftcards\Encryption\Factory\Factory;
+
+class StoreRegistryBuilder
+{
+    /**
+     * @var Factory
+     */
+    private $factory;
+
+    /**
+     * @var array
+     */
+    private $buildQueue = [];
+
+    /**
+     * Constructs a StoreRegistryBuilder with built-in StoreBuilders
+     * @param BuilderInterface[] $builders Additional builders
+     */
+    public function __construct($builders = [])
+    {
+        $builders[] = new DoctrineDBALStoreBuilder();
```

Comment:
can you put this into a static newInstance call then have the constructor just take a factory? That allows for a choice of what to put in the builder if they so choose